vis/js/components/README.md

```diff
@@ -0,0 +1,9 @@
+# Components
+React components. See https://reactjs.org/docs/components-and-props.html
+
+Each component should be stored in a separate file.
+
+Each component should have the same name as the file it is contained in.
+
+Each file should export both the plain component and have the redux connected
```

Comment:
Is this necessary? Usually if a component is connected to the store, it' used only with the store and never separately. In `Backlink.js`, the plain component is exported too, but that's only because I wasn't sure about how to write tests :grin: I stopped doing this in the next components (`ContextLine.js`, `Heading.js`).

Also, some components don't need Redux at all (`SubdisciplineTitle.js`).

Reply:
Sure; happy to make the change. I was also doing it to make testing easier :)